### PR TITLE
#16188 add setting to avoid extra actions comments for auto-generated DDLs

### LINKED ISSUE #16188: Feature wish: Option to disable automatic "source" comment on views
#### System information: 
- Operating system (distribution) and version: Microsoft Windows [Version 10.0.19043.1052]
- DBeaver version: Community 22.0.2
- Java version: As provided 
- Additional extensions: None

#### Describe the problem you're observing:
With some of the latest versions of DBeaver a feature was introduced, that the source editor window gets an additional "source" comment line automatically, like:
`-- SCHEMA.VIEW_NAME source`

We would like to be able to disable that, because working on that view duplicates that line over and over (or you have to remove that line manually, again and again).

While it was surely a nice idea, it does not help you much, if the view source looks like that:
```
-- SCHEMA.VIEW_NAME source

-- SCHEMA.VIEW_NAME source

CREATE OR REPLACE VIEW SCHEMA.VIEW_NAME AS 
...
```

#### Steps to reproduce, if exist:
Open "Source" tab of a view.

Thank you!

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2023-11-09 14_53_53-DBeaver Ultimate 23 2 4 - cb_session

2023-11-09 14_54_10-DBeaver Ultimate 23 2 4 - cb_session

vs

2023-11-09 14_53_10-DBeaver Ultimate 23 2 4 - cb_session

You can check view definitions in different databases. And, check PostgreSQL objects (like tables) and SQL Server with extended properties #12196 

All extra auto-generated info will be removed with this setting.

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java [hunk 1]
```diff
@@ -114,6 +121,7 @@ protected void loadPreferences(DBPPreferenceStore store)
                 SeparateConnectionBehavior.parse(store.getString(ModelPreferences.META_SEPARATE_CONNECTION))
             ));
             caseSensitiveNamesCheck.setSelection(store.getBoolean(ModelPreferences.META_CASE_SENSITIVE));
+            addExtraDDLInfo.setSelection(store.getBoolean(ModelPreferences.META_EXTRA_DDL_INFO));
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java:124]:
```suggestion
addExtraDDLInfo.setSelection(store.getBoolean(ModelPreferences.META_EXTRA_DDL_INFO));
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java:124]: I can't understand what is wrong here. Could you please explain it to me?

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java:124]: I see it with extra space
image

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java:124]: Oh, my bad, there should be space, the previous item is displayed wrong
github view confused me

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by uslss** [comment]
verified

**Pull request merged**